### PR TITLE
fix #79407 coin.mg

### LINKED ISSUE #79407: coin.mg
### Issue URL (Ads)
 [https://coin.mg/us1vsuBxJPY](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcoin.mg%2Fus1vsuBxJPY)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard Russian, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by wout-codes**
#79407

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -262,8 +262,9 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 downloads-anymovies.com###e15
+coin.mg##div.col-md-12:not(#go-link)
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:265]: Why did you add `:not(#go-link)`?

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:265]: Blocks the link in the final page.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:265]: @zloyden

Comment (zloyden) [reply EnglishFilter/sections/specific.txt:265]: Can you provide a screenshot with this element? I mean with this additional `id`, can't find it.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:265]: I can't visit the site anymore. The link died. @Yuki2718 Can you bail me out?

Comment (Yuki2718) [reply EnglishFilter/sections/specific.txt:265]: I can't either.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:265]: @zloyden I guess you should just trust me, or check it out yourself.

Comment (Alex-302) [reply EnglishFilter/sections/specific.txt:265]: `coin.mg##div.col-md-12:not(#go-link)` is dangerous, and break this page
`https://coin.mg/bonus/dogecoin/?r=DG9EJ8JBUdupdoZVqcPcCN1dFr3AnrNwaH

**Review by Alex-302 — CHANGES_REQUESTED** [review]
